packages/common/test/events/_internal/EventSchemaEdit.test.ts

```diff
@@ -73,6 +73,9 @@ describe("EventSchemaEdit", () => {
             type: "boolean",
             default: false,
           },
+          timeZone: {
```

Comment:
In order for the `required` error message to display from the UISchema definition, I _believe_ we need to add `timeZone` to the array of `required` fields at the top of this schema.